### ISSUE TITLE
Draft Top 10 Item: Missing Trust Boundaries

### ISSUE THREAD

**Issue by andreashappe** [OWASP/www-project-operational-technology-top-10#14]
To test the prototype template at https://ot.owasp.org/the-top-10/template/

Include: missing network separation, but also think about trust boundaries in other systems (e.g., medical devices)

**Comment by andreashappe** [comment]
for easier sync, there's now a branch at: https://github.com/OWASP/www-project-operational-technology-top-10/tree/item-trust-boundaries

**Comment by simonrommer** [comment]
link shows 404

**Comment by andreashappe** [comment]
true, please create a new branch (or directly create a pul request against `main`). Basically the branches became very outdated when I split up the top 10 items into separate files (and to prevent duplicate work, I removed them).
